### PR TITLE
Removed Header Test, Added Hello Logo Test

### PR DESCRIPTION
**Pull request by daveccrit**
Removed Header tests since header in loop has been removed. Added test for Firefox Hello logo, as that was added.

### CODE REVIEW

**Comment by rbillings** [comment]
Local tests fail as test_supported_browser is looking for the header_title_text instead of the updated is_hello_logo_visible

File: tests/test_hello.py [hunk 1]
```diff
@@ -14,7 +14,7 @@ class TestHelloPage:
     def test_supported_browser(self, mozwebqa):
         hello_page = HelloPage(mozwebqa)
         hello_page.go_to_page()
-        assert hello_page.is_header_title_visible
+        assert hello_page.is_hello_logo_visible
         assert hello_page.is_invalid_conversation_visible
         assert hello_page.is_mozilla_logo_visible
         assert 'Firefox Hello' == hello_page.header_title_text
```
Comment (davehunt) [inline tests/test_hello.py]: This assert needs to be removed as the `header_title_text` property was removed from the page object.

**Comment by dmose** [comment]
How is it possible that local tests were failing while remote ones were not?  Is the test setup somehow buggy?

**Comment by davehunt** [comment]
@dmose which remote tests are passing? I can't see how these tests would pass as they are.

**Comment by dmose** [comment]
Never mind; I was confused.

**Pull request merged**